packages/cli/src/concurrency/concurrency-control.service.ts

```diff
@@ -34,6 +33,8 @@ export class ConcurrencyControlService {
 		private readonly telemetry: Telemetry,
 		private readonly eventService: EventService,
 	) {
+		this.logger = this.logger.withScope('executions');
```

Comment:
It'd be nice to make this into a class or constructor parameter decorator. I tried this but it interferes with `@Service` from `typedi` so leaving it for the future.